xml/System/Console.xml

```diff
@@ -5666,6 +5666,8 @@
 ## Examples  
  The following example demonstrates the standard formatting specifiers for numbers, dates, and enumerations.  
   
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]
```

Comment:
the note should be included in the top of the page instead (membergroup element remarks - line 4725)

Reply:
Done.